include/openssl/ssl.h.in

```diff
@@ -1401,6 +1402,8 @@ DECLARE_PEM_rw(SSL_SESSION, SSL_SESSION)
         SSL_ctrl(s,SSL_CTRL_SET_CHAIN_CERT_STORE,1,(char *)(st))
 # define SSL_get1_groups(s, glist) \
         SSL_ctrl(s,SSL_CTRL_GET_GROUPS,0,(int*)(glist))
+# define SSL_get0_iana_groups(s, plst) \
+        SSL_ctrl(s, SSL_CTRL_GET_IANA_GROUPS, 0, plst)
```

Comment:
We may want to cast `plst` here, since we are effectively using it as an `uint16_t *`.

Also, bizarrely, the prevailing style here has no space after comma.

Reply:
done.